docs/UsersGuide/source/VXCases.rst

```diff
@@ -0,0 +1,301 @@
+
+   * `Storm Prediction Center Storm Report for 20190615 <https://www.spc.noaa.gov/climo/reports/190615_rpts.html>`__
+   * `Storm Prediction Center Storm Report for 20190616 <https://www.spc.noaa.gov/climo/reports/190616_rpts.html>`__
+
+Set Up Verification
+======================
+
+Follow the instructions below to reproduce a forecast for this event using your own model setup! Make sure to install and build the latest version of the SRW Application (v2.1.0). ``develop`` branch code is constantly changing, so it does not provide a consistent baseline for comparison. 
+
+.. _GetSampleData:
+
+Download Data
+----------------
+
+Download the ``Indy-Severe-Weather.tgz`` file using any of the following methods: 
+
+   #. Download directly from the S3 bucket using a browser. The data is available at https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#sample_cases/release-public-v2.1.0/.
+
+   #. Download from a terminal using the AWS command line interface (CLI) if installed:
+
+      .. code-block:: console
+
+         aws s3 cp https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#sample_cases/release-public-v2.1.0/Indy-Severe-Weather.tgz Indy-Severe-Weather.tgz
+   
+   #. Download from a terminal using wget: 
+
+      .. code-block:: console
+
+         wget https://noaa-ufs-srw-pds.s3.amazonaws.com/sample_cases/release-public-v2.1.0/Indy-Severe-Weather.tgz
+
+This tar file contains :term:`IC/LBC <IC/LBCs>` files, observation data, model/forecast output and MET verification output for the sample forecast. Users who have never run the SRW App on their system before will also need to download the fix files required for SRW App forecasts and the NaturalEarth shapefiles required for plotting. Users can download the fix file data from a browser at https://noaa-ufs-srw-pds.s3.amazonaws.com/current_srw_release_data/fix_data.tgz or visit :numref:`Section %s <StaticFixFiles>` for instructions on how to download the data with wget. NaturalEarth files are available at https://noaa-ufs-srw-pds.s3.amazonaws.com/NaturalEarth/NaturalEarth.tgz. See the :ref:`Graphics <srw_v2.1.0:Graphics>` chapter of the release documentation for more information. 
+
+After downloading ``Indy-Severe-Weather.tgz`` using one of the three methods above, untar the downloaded compressed archive file: 
+
+.. code-block:: console
+
+   tar xvfz Indy-Severe-Weather.tgz
+
+Record the path to this file output using the ``pwd`` command: 
+   
+.. code-block:: console 
+
+   cd Indy-Severe-Weather
+   pwd
+
+.. note::
+
+   Users can untar the fix files and Natural Earth files by substituting those file names in the commands above. 
+
+Load the Regional Workflow
+-----------------------------
+
+First, navigate to the ``ufs-srweather-app/ush`` directory. Then, load the regional workflow environment:
+
+.. code-block:: console
+   
+   source <path/to/etc/lmod-setup.sh>
+   module use </path/to/ufs-srweather-app/modulefiles>
+   module load wflow_<platform>
+
```

Comment:
Should either:

`conda activate regional_workflow`

or

`Please copy the command shown after loading the wflow_<platform> modulefile to load the regional_workflow conda environment.`

also be added to the end of this list of commands to load the regional workflow conda environment (at least on Level 1 platforms)?